tcms/testruns/tests/test_admin.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+
+
+from django.urls import reverse
+
+from tcms.tests import LoggedInTestCase
+
+
+class TestTestRunAdmin(LoggedInTestCase):
+
+    def test_regular_user_can_not_add_view(self):
+        response = self.client.get(reverse('admin:testruns_testrun_add'))
+        self.assertRedirects(response, reverse('admin:testruns_testrun_changelist'))
```

Comment:
This is failing with HTTP response code 403 (Forbidden) which means you don't have permissions to access the requested page.

`self.tester` is logged in but is not allowed to access admin pages.

Define a `setUpTestData()` method which sets `cls.tester.is_staff = True` and saves the object. This should allow the request to reach the admin page.